### PR TITLE
Some tweaks

### PR DESCRIPTION
**Pull request by bcvandendool**
* Rename stuff in resources page
* Change logo on front page
* Change committee image  ratio to 1:1

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/users/models/committee.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 
 def validate_aspect_ratio(image):
-    ratio = 16 / 9
+    ratio = 1 / 1
```
Comment (dasyad00) [inline apps/users/models/committee.py:9]: what happens to old committee photos currently in the db? do all of them throw ValidationErrors(s) or is it only when the db gets changed?

File: cosmos/templates/resources.html [hunk 2]
```diff
@@ -56,15 +56,15 @@ <h5 class="text-center mt-5">Discord</h5>
         <a class="btn btn-cosmos-primary" href="https://discord.gg/pep5FZ9">Join our Discord</a>
     </div>
 
-    <h5 class="text-center mt-5">Policy Documents and GMM Minutes</h5>
+    <h5 class="text-center mt-5">Internal Documents</h5>
     <p>
-        The Cosmos policy documents and GGM minutes can be found on the pages linked
+        The Cosmos policy documents and GGM documents can be found on the pages linked
```
Comment (dasyad00) [inline cosmos/templates/resources.html:61]: typo "GGM"

File: cosmos/templates/index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <section class="home-img min-vh-100">
     <div class="container d-flex h-100">
         <div class="mx-auto text-center">
-            <img class="img-fluid mh-50" src="{% static 'cosmos/img/cosmos_white_horizontal.png' %}">
+            <img class="img-fluid mh-50" src="{% static 'cosmos/img/Cosmos_alternate_desriptor.svg' %}">
```
Comment (dasyad00) [inline cosmos/templates/index.html:5]: minor complaint: can the name of the file be lowercase?

**Review by dasyad00 — CHANGES_REQUESTED** [review]

File: apps/users/models/committee.py (hunk 1 above)
Comment (bcvandendool) [reply apps/users/models/committee.py:9]: only when you want to change it will it matter

**Review by dasyad00 — APPROVED** [review]

**Pull request merged**